### PR TITLE
JAVA_HOME env var

### PR DESCRIPTION
**Pull request by hochgi**

### CODE REVIEW

File: server/cmwell-cons/app/cmw.env [hunk 1]
```diff
@@ -1,6 +1,7 @@
 export PATH=`pwd`/components-extras/scala/bin:$PATH
 export PATH=`pwd`/components-extras/java/bin:$PATH
 export PATH=`pwd`/bin/utils:$PATH
+export JAVA_HOME=`pwd`/components-extras/java
```
Comment (e-orz) [inline server/cmwell-cons/app/cmw.env]: cm-well doesn't always deploy itself java.
Changing this blindly will cause systems without deployed java to have a wrong JAVA_HOME

Comment (hochgi) [reply server/cmwell-cons/app/cmw.env]: You're absolutely right.
What do you suggest instead then...?

Comment (bryaakov) [reply server/cmwell-cons/app/cmw.env]: Just a thought, "getOrElseExport" will do the trick here?
Something like ```[ -z "$JAVA_HOME" ] && export JAVA_HOME=`pwd`/components-extras/java```

Comment (hochgi) [reply server/cmwell-cons/app/cmw.env]: Problem is with servers that has env var defined to old java, and it takes precedence over PATH.

Comment (e-orz) [reply server/cmwell-cons/app/cmw.env]: The simple solution will be to check whether java is deployed with cm-well and only then set the JAVE_HOME

**Pull request closed without merging**